packages/now-cli/test/dev-server.unit.js

```diff
@@ -175,6 +175,26 @@ test(
     t.is(parsed.query['route-param'], 'b');
   })
 );
+test(
+  '[DevServer] Maintains query when builder defines routes through rewrites',
+  testFixture('now-dev-next-rewrites', async (t, server) => {
```

Comment:
`now-dev` should be `vc-dev`

Reply:
I hesitated but since there is not a single fixture named `vc-dev` in the unit tests I mistakenly assumed that consistency was better.